### DISCUSSION TITLE
New admin option for custom component at top of page

### DISCUSSION THREAD

**Discussion by JessRynkar** [payloadcms/payload#7584] [Feature Requests & Ideas]
Add new admin option that allows the user to insert a custom component at the top of the page globally (similar to beforeNavLinks / afterNavLinks).

Potential option names:
- topbar
- header
- any other ideas?

PR: https://github.com/payloadcms/payload/pull/7647

**Comment by mikecebul** [comment]
Maybe appbar, but I think I prefer header. Too bad it couldn't be Nextjs specific cause you could just use a root layout which we could use for a header, footer, and/or sidebar.
